core/src/ast_builder/mod.rs

```diff
@@ -77,13 +77,51 @@ pub use expr::{
     },
 };
 
+use {self::table::TableType, crate::ast::Dictionary};
 /// Entry point function to build statement
 pub fn table(table_name: &str) -> TableNode {
     let table_name = table_name.to_owned();
 
-    TableNode { table_name }
+    TableNode {
+        table_name,
+        table_type: TableType::Table,
+    }
 }
 
+pub fn glue_objects<'a>() -> TableNode<'a> {
+    TableNode {
+        table_name: "GLUE_OBJECTS".to_owned(),
+        table_type: TableType::Dictionary(Dictionary::GlueObjects),
+    }
+}
+
+pub fn glue_tables<'a>() -> TableNode<'a> {
+    TableNode {
+        table_name: "GLUE_TABLES".to_owned(),
+        table_type: TableType::Dictionary(Dictionary::GlueTables),
+    }
+}
+
+pub fn glue_indexes<'a>() -> TableNode<'a> {
+    TableNode {
+        table_name: "GLUE_INDEXES".to_owned(),
+        table_type: TableType::Dictionary(Dictionary::GlueIndexes),
+    }
+}
+
+pub fn glue_table_columns<'a>() -> TableNode<'a> {
+    TableNode {
+        table_name: "GLUE_TABLE_COLUMNS".to_owned(),
+        table_type: TableType::Dictionary(Dictionary::GlueTableColumns),
+    }
+}
```

Comment:
1. nit. except for `series` which takes `Into<ExprNode<'a>>`, all has static lifetimes.

e.g.
```rust
pub fn glue_table_columns() -> TableNode<'static> { .. }
```
this would be enough.

---
2. Now table sugar functions are more than a single one, then how about moving these including `series` into `table.rs` module?